<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plaintext</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PublicKey</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PrivateKey</string>
    </property>
   </widget>
   <widget class="QLabel" name="PublicKey_Content">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>******************</string>
    </property>
   </widget>
   <widget class="QLabel" name="PrivateKey_Content">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>******************</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Plaintext_Content">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="Ciphertext_Content">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ciphertext</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Plaintext</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="New_Plaintext_Content">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="Generate_Key_Button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Enctypt_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dectypt_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
